source/javascripts/components/_AddStep.js.erb

```diff
@@ -463,14 +461,32 @@ angular.module("BitriseWorkflowEditor").service("addStepService", function() {
 		}
 	};
 
-}).directive("addStepSidebarStep", function($animate) {
+}).directive("addStepSidebarStep", function($filter, addStepService, imageService) {
```

Comment:
can we add injection names? like `["$filter", "addStepService", "imageService", function($filter, addStepService, imageService) {...}]`? that's the only way if we want to minify it later etc, that's the best practice.

Reply:
I'd rather do it separately, we currently use this format for dependency injection everywhere in the Workflow Editor. In a separate PR we could collectively replace all the dependency injections to fix the issue. I created an internal maintenance task for it.